router/core/http_server.go

```diff
@@ -44,13 +50,20 @@ func newServer(opts *httpServerOptions) *server {
 		MaxHeaderBytes: opts.maxHeaderBytes,
 	}
 
+	// Create default handler for liveness and readiness
+	httpRouter := chi.NewMux()
```

Comment:
When we add them here, don't we need to remove them from `newGraphServer`?

Reply:
No because we are creating a completely new mux and replace the existing one in the Swap function.